tests/common/util/TestObject.cpp

```diff
@@ -330,13 +330,16 @@ createMetaDataForBookChange(
     int finalTakerGets,
     int perviousTakerGets,
     int finalTakerPays,
-    int perviousTakerPays
+    int perviousTakerPays,
+    std::optional<std::string_view> const& domain
```

Comment:
nit: Maybe we should just pass by copy since it's a string_view inside of optional?